README.md

```diff
@@ -97,20 +114,20 @@ The following bootstrap playbook requires a number of configs to be passed in, n
 * `tower_environment`: The Ansible Tower environment (dev/test/qe)
 
 ```bash
-ansible-playbook -i inventories/hosts playbooks/bootstrap_tower.yml -e tower_host=<tower-host> -e tower_username=<tower-username> -e tower_password=<tower-password> -e tower_environment=<tower-environment>
+ansible-playbook -i <path-to-local-credentials-project>/inventories/hosts playbooks/bootstrap_tower.yml -e tower_host=<tower-host> -e tower_username=<tower-username> -e tower_password=<tower-password> -e tower_environment=<tower-environment>
```

Comment:
No need to specify host, verify_ssl, tower_username or tower_password if you have a creds repo.